_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_althland_facility.dmm

```diff
@@ -915,7 +915,8 @@
 	},
 /obj/machinery/mineral/processing_unit_console{
 	pixel_y = -2;
-	pixel_x = -29
+	pixel_x = -29;
+	density = 0
```

Comment:
Change this on the /obj/machinery/mineral/processing_unit_console definition instead of var-editing it on the map, as I believe this should never be dense